index.js

```diff
@@ -132,7 +132,8 @@ module.exports = {
 
   postprocessTree: function(type, tree) {
     if (type === 'all' || type === 'styles') {
-      tree = autoprefixer(tree, { browsers: ['last 2 versions'] });
+      tree = autoprefixer(tree,
+          this.app.options.autoprefixer || { browsers: ['last 2 versions'] });
```

Comment:
I'd be more in favor of explicit browser support in the autoprefixer, rather than a rule that's interpretation changes as new browser versions are released. Possibly something like `['Chrome >= 50', 'Edge >= 12', 'ff >= 46', 'ie >= 9', 'ie_mob >= 10', 'ios_saf >= 8.1', 'Opera >= 37', 'Safari >= 8']
` which is the translation of `['last 3 versions']` that @vine77 listed. This way dropping and adding support is done intentionally and doesn't cause any surprises down the road.